main_app/templates/home.html

```diff
@@ -3,36 +3,70 @@
 {% block content %}
 
 {% if user.is_authenticated %}
-<div class="col-md-6">
-  <h1>Welcome to IVM</h1>
```

Comment:
Good decision to take away the h1 here. The header already has the app's name on it on the nav bar, so we can just keep the focus of the body on the actual content. Good job.

Reply:
Agreed, Nav Bar does the job for this, good job!